build-system/pr-check/checks.js

```diff
@@ -59,6 +59,7 @@ async function main() {
     timedExecOrDie('gulp dep-check');
     timedExecOrDie('gulp check-types');
     timedExecOrDie('gulp check-sourcemaps');
+    timedExecOrDie('gulp report-test-results');
```

Comment:
This is very much not the right file for this; I was following the owners-check task too literally. I'll move this to somewhere more suitable.

Reply:
Done.